ajax/libs/dio/package.json

```diff
@@ -31,7 +31,8 @@
       "basePath": "/",
       "files": [
         "dio.min.js",
-        "dio.js"
+        "dio.js",
+        "dio*"
```

Comment:
Yes the latest [commit](https://github.com/cdnjs/cdnjs/pull/8770/commits/dc451ddf8c7661bcefd46fd310708002d453325c) implements that change.